test/parseValue-test.js

```diff
@@ -118,6 +169,11 @@ const tests = [
   [ '(1234.56$)', -1234.56, '#,##0.00$' ],
   [ '( 1234.56 $ )', -1234.56, '#,##0.00$' ],
   [ '((1234.56$))', null, null ],
+  
+  // multiple currency symbols in input
+  [ '$$1234', null, '$#,##0' ],
+  [ '$€1234', null, '$#,##0' ],
+  [ '€$1234', null, '€#,##0' ],
```

Comment:
These are the weird test cases referred to in the description. I'm not sure if numfmt should:
1. not return numfmt if multiple currency symbols in string
2. do a better job at parsing the value but return the numfmt as shown in the test case

Reply:
I think that numfmt should not parse these as currency values (or at least not be default). I've had a look at this and it turns out that the test was bugged: https://github.com/borgar/numfmt/commit/605585a4141e74a8e0285f56850e23ac0c2de0c6